### DISCUSSION TITLE
How do we search?

### DISCUSSION THREAD

**Discussion by bondijoe27** [microsoft/graphrag#821] [Q&A] [answered]
I have checked the dataflow and explanation of local search and global search. of the GraphRAG from https://microsoft.github.io/graphrag/posts/index/1-default_dataflow/. And I have questions regarding it. From the dataflow, it seems tha t communities are constructed from each document, i.e. communities do not interact with each other if they are from different documents. So when you search for an answer, the results must come from one single document and it cannot merge results from different documents?

**Comment by LIHUA919** [comment]
I donot know

**Comment by natoverse** [comment, marked as answer]
Communities actually come from the entity graph extracted from the documents. Entities in the same document may not end up in the same community, so document content can in that way be spread among many communities.

**Reply by bondijoe27** [reply]
So I guess if I have a lot of documents as my input, elements (or nodes) from different documents can form a community?
